commit b58d6551c319c168220a5ee13af33be67cf52e70
Author: Steven Fackler <sfackler@gmail.com>
Date:   Tue Jun 7 19:22:20 2022 -0400

    Propagate CXXFLAGS_FOR_BUILD to sub-makes

diff --git a/Makefile.def b/Makefile.def
index 72d58549645..07b61063cf6 100644
--- a/Makefile.def
+++ b/Makefile.def
@@ -261,6 +261,7 @@ flags_to_pass = { flag= BISON ; };
 flags_to_pass = { flag= CC_FOR_BUILD ; };
 flags_to_pass = { flag= CFLAGS_FOR_BUILD ; };
 flags_to_pass = { flag= CXX_FOR_BUILD ; };
+flags_to_pass = { flag= CXXFLAGS_FOR_BUILD ; };
 flags_to_pass = { flag= EXPECT ; };
 flags_to_pass = { flag= FLEX ; };
 flags_to_pass = { flag= INSTALL ; };
diff --git a/Makefile.in b/Makefile.in
index 593495e1650..b01fece2e04 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -842,6 +842,7 @@ BASE_FLAGS_TO_PASS = \
 	"CC_FOR_BUILD=$(CC_FOR_BUILD)" \
 	"CFLAGS_FOR_BUILD=$(CFLAGS_FOR_BUILD)" \
 	"CXX_FOR_BUILD=$(CXX_FOR_BUILD)" \
+	"CXXFLAGS_FOR_BUILD=$(CXXFLAGS_FOR_BUILD)" \
 	"EXPECT=$(EXPECT)" \
 	"FLEX=$(FLEX)" \
 	"INSTALL=$(INSTALL)" \
